<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddStudentDialog</class>
 <widget class="QDialog" name="AddStudentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#FFF3E6</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>学号：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>姓名：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineName">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    background-color: #e6f7ff;
    border: 2px solid #91d5ff;
    border-radius: 6px;
    padding: 6px;
    font-size: 15px;
    color: #004b6e;
}
QLineEdit:focus {
    border: 2px solid #40a9ff;
    background-color: #ffffff;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>语文成绩：</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>数学成绩：</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>英语成绩：</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineMath">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    background-color: #e6f7ff;
    border: 2px solid #91d5ff;
    border-radius: 6px;
    padding: 6px;
    font-size: 15px;
    color: #004b6e;
}
QLineEdit:focus {
    border: 2px solid #40a9ff;
    background-color: #ffffff;
}</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEnglish">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    background-color: #e6f7ff;
    border: 2px solid #91d5ff;
    border-radius: 6px;
    padding: 6px;
    font-size: 15px;
    color: #004b6e;
}
QLineEdit:focus {
    border: 2px solid #40a9ff;
    background-color: #ffffff;
}</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineChinese">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    background-color: #e6f7ff;
    border: 2px solid #91d5ff;
    border-radius: 6px;
    padding: 6px;
    font-size: 15px;
    color: #004b6e;
}
QLineEdit:focus {
    border: 2px solid #40a9ff;
    background-color: #ffffff;
}</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineID">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    background-color: #e6f7ff;
    border: 2px solid #91d5ff;
    border-radius: 6px;
    padding: 6px;
    font-size: 15px;
    color: #004b6e;
}
QLineEdit:focus {
    border: 2px solid #40a9ff;
    background-color: #ffffff;
}</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #fbb1bd;  /* 草莓粉 */
    color: white;
    font-weight: bold;
    font-size: 15px;
    border: 2px solid #fda4af;
    border-radius: 12px;
    padding: 8px 16px;
}
QPushButton:hover {
    background-color: #ffa8a8;  /* 悬停颜色 */
}
QPushButton:pressed {
    background-color: #ff8c94;  /* 点击颜色 */
}</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineID</tabstop>
  <tabstop>lineName</tabstop>
  <tabstop>lineChinese</tabstop>
  <tabstop>lineMath</tabstop>
  <tabstop>lineEnglish</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddStudentDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddStudentDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
